conf/documentation.conf

```diff
@@ -573,6 +573,12 @@ description=<<EOT
 The role to assign to devices registered from the specific portal. If none is specified, the role of the registrant is used.
 EOT
 
+[device_registration.oses]
+type=fingerbank_select
+description=<<EOT
+Lists of OSES where the MAC vendor will be allowed to be registered via the device registration page.
```

Comment:
Shouldn't we just say "List of allowed devices to be registered via the device registration page." ? If we use Fingerbank's device, we don't need to tie this with OS/MAC vendors.